productcomments.php

```diff
@@ -828,12 +828,10 @@ public function hookHeader()
     {
         $jsList = [];
         $cssList = [];
-        if ($this->context->controller instanceof ProductControllerCore ||
-            $this->context->controller instanceof ProductListingFrontControllerCore ||
-            $this->context->controller instanceof IndexControllerCore) {
```

Comment:
Why do you remove these lines?

Reply:
Well, we should be able to load product ratings everywhere. Why should it to be limited only to listing pages, index and product page, since we can use product listing on any page (for example via widgets).